index.js

```diff
@@ -27,10 +27,10 @@ SwigWebpackPlugin.prototype.apply = function(compiler) {
 		var context = this.context;
 
 		if (self.options.watch) {
-			var watchFiles = glob.sync(self.options.watch, {});
+			var watchFiles = glob.sync(path.join(context, self.options.watch), {});
 			if(watchFiles.length > 0) {
 				watchFiles.forEach(function(file) {
-					compiler.fileDependencies.push(path.join(context, file));
+					compiler.fileDependencies.push(file);
```

Comment:
there is one problem with this line:
I'm working on windows, therefore `path.join` outputs a string with backslashes on my system. `glob` is working with forward slashes only (https://github.com/isaacs/node-glob#windows). so this line does not work on my windows machine, because the string in `file` has forward slashes.
can you please check if you can solve your context issues with one of the `glob` options? https://github.com/isaacs/node-glob#options
thanks!